src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -42,6 +42,8 @@
 
 # Profile datatype is used by ConfigFile to return Profile Data along with
 # metadata such as profile_name and secure_props_not_found
+
+
 class Profile(NamedTuple):
```

Comment:
For the `autodiscover_config_dir` function, do we want to skip over `zowe.config.user.json` if it does not exist? It is optional, so we should avoid throwing exceptions for a missing user-level config.

Reply:
This could be resolved by implementing a try-except block while invoking the function.